src/components/trezor/SendBitcoinTrezor.vue

```diff
@@ -0,0 +1,267 @@
+<template>
```

Comment:
this seems like a mistake... you are readding the removed sendbitcointrezor component